common/types.go

```diff
@@ -425,6 +453,8 @@ which is sufficient data to set the bid of the builder. The `Transactions`
 and `Withdrawals` fields are required to construct the full SignedBeaconBlock
 and are parsed asynchronously.
 
+TODO(JWT): Does this need to be updated? It hasn't been updated for Deneb.
```

Comment:
can be separate upgrade added deneb support here https://github.com/flashbots/mev-boost-relay/pull/618

Reply:
Yes, good idea. Thank you for making that PR. I will remove this comment.